### PR TITLE
GraphQL Playground is added

### PR DESCRIPTION
**Pull request by AndrewOlenik**
Introduce GraphQL Playground #24

### CODE REVIEW

File: PLAYGROUND.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Intro
+
+Voyager sample contains:
```
Comment (trioletas) [inline PLAYGROUND.md]: `Voyager` -> `Playground`

File: PLAYGROUND.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Requirements:
+  * Java 1.8
+  * Spring Framework Boot > 1.5.7 (web)
+  * runtime voyager dependencies are loaded from `//cdn.jsdelivr.net`
+
+# Enable GraphQL Playground
+
+GraphQL Playground becomes accessible at the root `/playground`.
+
+Note that GraphQL server must be available at `/graphql/*` context to be discovered by GraphQL Playground.
```
Comment (trioletas) [inline PLAYGROUND.md]: that's a default behavior that can be overriden by the `plaground.endpoint` configuration but from the wording of that sentence it looks like a hard constraint
perhaps it makes sense to change the wording to something like
```
Note that by default, the GraphQL server must be available at `/graphql/*` endpoint to be discovered by GraphQL Playground. 
```

File: PLAYGROUND.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+Voyager sample contains:
+* `playground.html` the home page of [GraphQL Playground](https://github.com/graphcool/graphql-playground) - GraphQL IDE for better development workflows (GraphQL Subscriptions, interactive docs & collaboration).
+* `lv.ctco.tpl.bff.playground` configures and serves playground static content using Web Controller
+
+# Requirements and Downloads
+
+Requirements:
+  * Java 1.8
+  * Spring Framework Boot > 1.5.7 (web)
+  * runtime voyager dependencies are loaded from `//cdn.jsdelivr.net`
```
Comment (trioletas) [inline PLAYGROUND.md]: `voyager` -> `playground`

File: api/src/main/java/lv/ctco/tpl/bff/playground/PlaygroundConfiguration.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import org.springframework.web.servlet.DispatcherServlet;
+
+@Configuration
+@ConditionalOnWebApplication
+@ConditionalOnClass(DispatcherServlet.class)
+public class PlaygroundConfiguration {
+
+    @Bean
+    @ConditionalOnProperty(value = "playground.enabled", havingValue = "true", matchIfMissing = true)
+    PlaygroundController voyagerController() {
```
Comment (trioletas) [inline api/src/main/java/lv/ctco/tpl/bff/playground/PlaygroundConfiguration.java]: `voyager` -> `playground`

File: api/src/main/java/lv/ctco/tpl/bff/playground/PlaygroundController.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+public class PlaygroundController {
+
+    @Value("${playground.endpoint:/graphql}")
+    private String graphqlEndpoint;
+
+    @Value("${playground.pageTitle:Playground}")
+    private String pageTitle;
+
+    @RequestMapping(value = "${playground.mapping:/playground}")
+    public void voyager(HttpServletResponse response) throws IOException {
```
Comment (trioletas) [inline api/src/main/java/lv/ctco/tpl/bff/playground/PlaygroundController.java]: `voyager` -> `playground`

File: PLAYGROUND.md (hunk 1 above)
Comment (AndrewOlenik) [reply PLAYGROUND.md]: thx for spotting

**Pull request merged**